.github/workflows/coverage.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: 
       - codecov
```

Comment:
There is no "codecov" branch. Change this to "main" if you are merging this into main.